core/src/themes/ionic.globals.ionic.scss

```diff
@@ -0,0 +1,209 @@
+
+$ionic-display-m:         dynamic-font(36px);
+$ionic-display-s:         dynamic-font(32px);
+
+$ionic-h1-font-size:      dynamic-font(28px) !default;
+$ionic-h2-font-size:      dynamic-font(26px) !default;
+$ionic-h3-font-size:      dynamic-font(24px) !default;
+$ionic-h4-font-size:      dynamic-font(22px) !default;
+$ionic-h5-font-size:      dynamic-font(20px) !default;
+$ionic-h6-font-size:      dynamic-font(18px) !default;
+
+$ionic-font-size-s:       dynamic-font(12px);
+$ionic-font-size-base:    dynamic-font(14px);
+$ionic-font-size-l:       dynamic-font(16px);
+
+:root {
+  /* Font Family */
+  --ionic-font-family: "Inter", sans-serif;
+
+  /* Font sizes */
+  --ionic-font-size-s: #{$ionic-font-size-s};
+  --ionic-font-size-base: #{$ionic-font-size-base};
+  --ionic-font-size-l: #{$ionic-font-size-l};
+
+  /* Font size displays */
+  --ionic-display-s: #{$ionic-display-s};
+  --ionic-display-m: #{$ionic-display-m};
+  --ionic-line-height-display-s: 44px;
+  --ionic-line-height-display-m: 48px;
+
+  /* Headings */
+  --ionic-h1-font-size: #{$ionic-h1-font-size};
+  --ionic-h2-font-size: #{$ionic-h2-font-size};
+  --ionic-h3-font-size: #{$ionic-h3-font-size};
+  --ionic-h4-font-size: #{$ionic-h4-font-size};
+  --ionic-h5-font-size: #{$ionic-h5-font-size};
+  --ionic-h6-font-size: #{$ionic-h6-font-size};
+
+  /* Font Weights */
+  --ionic-font-light: 300;
+  --ionic-font-regular: 400;
+  --ionic-font-medium: 500;
+  --ionic-font-semibold: 600;
+  --ionic-font-bold: 700;
+
+  /* Line heights */
+  --ionic-line-height-xs: 16px;
+  --ionic-line-height-s: 20px;
+  --ionic-line-height-m: 24px;
+  --ionic-line-height-l: 28px;
+  --ionic-line-height-xl: 32px;
+  --ionic-line-height-xxl: 36px;
+
+  /* Letter spacing */
+  --ionic-letter-spacing-none: 0em;
+  --ionic-letter-spacing-s: 0.1em;
+  --ionic-letter-spacing-m: 0.15em;
+}
+
+html {
+  font-family: var(--ionic-font-family);
+  font-size: var(--ionic-font-size-base);
+  font-weight: var(--ionic-font-regular);
+  line-height: var(--ionic-line-height-m);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: var(--ionic-color-base);
+  font-weight: var(--ionic-font-semibold);
+}
+
+h1 {
+  font-size: var(--ionic-h1-font-size);
+  line-height: var(--ionic-line-height-xxl);
+}
+
+h2 {
+  font-size: var(--ionic-h2-font-size);
+  line-height: var(--ionic-line-height-xl);
+}
+
+h3 {
+  font-size: var(--ionic-h3-font-size);
+  line-height: var(--ionic-line-height-l);
+}
+
+h4 {
+  font-size: var(--ionic-h4-font-size);
+  line-height: var(--ionic-line-height-l);
+}
+
+h5 {
+  font-size: var(--ionic-h5-font-size);
+  line-height: var(--ionic-line-height-l);
+}
+
+h6 {
+  font-size: var(--ionic-h6-font-size);
+  line-height: var(--ionic-line-height-s);
+}
+
+/* Utility Classes */
+/* Headings */
+.ionic-heading1,
+.ionic-heading2,
+.ionic-heading3,
+.ionic-heading4,
+.ionic-heading5,
+.ionic-heading6 {
+  color: var(--ionic-color-base);
+  font-weight: var(--ionic-font-semibold);
```

Comment:
Why is font-weight applied to all the classes but only to the h1 tag?

Reply:
I can't understand the question because the `font-weight` is equal to all heading styles, `semibold`. To the others will be the `regular`.

Reply:
Apologies on this one! I didn't see it correctly at the time. My comment is irrelevant.